docker/monitoring/prometheus/prometheus.yml

```
@@ -0,0 +1,20 @@
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['localhost:9090']
+
+  - job_name: 'ururu-backend'
+    static_configs:
+      - targets: ['3.39.69.34:80']
+    metrics_path: '/actuator/prometheus'
+    scrape_interval: 10s
+    scrape_timeout: 5s
+    honor_labels: true 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)